target/linux/ipq40xx/image/Makefile

```diff
@@ -158,6 +158,22 @@ define Device/engenius_eap1300
 endef
 TARGET_DEVICES += engenius_eap1300
 
+define Device/engenius_ens620ext
+        $(call Device/FitImage)
+        DEVICE_DTS := qcom-ipq4018-ens620ext
+        DEVICE_DTS_CONFIG := config@4
+        BLOCKSIZE := 128k
+        PAGESIZE := 2048
+        DTB_SIZE := 65536
+        DEVICE_TITLE := EnGenius ENS620EXT
+        IMAGE_SIZE := 20439364
+        KERNEL_SIZE := 5120k
```

Comment:
Is this limit something the u-boot loader enforces? Can you please provide a "printenv" from the u-boot prompt / or a dump of your u-boot-env partition. Thanks.

Reply:
printenv: 
app_part=0
athaddr=88:dc:96:00:00:13
baudrate=115200
bootcmd=bootipq
bootdelay=2
bundle_ap_mac=88:dc:96:00:00:15
country=000
ddwrt_sw=0
debug=0
domain=0
enable_failsafe=0
ethact=eth0
ethaddr=88:dc:96:00:00:10
fwaddr=88:dc:96:00:00:14
hw_id=00000000
hw_ver=0.0.1
ipaddr=192.168.1.1
language_code=00
machid=8010000
oled_on=0
op_mode=0
pro_id=000
serverip=192.168.1.37
service_tag=000000000000000
sn=000000001
snextra=00000000000000000000
stderr=serial
stdin=serial
stdout=serial
uboot_ver=1.0.0.1
wanaddr=88:dc:96:00:00:11
wlanaddr=88:dc:96:00:00:12

Reply:
Thank you.

```bootcmd=bootipq```

so you can leave it as is.